src/cloud/project/new-relic.md

```diff
@@ -175,7 +208,42 @@ All collected data details the time spent on an action transmitting data, querie
 {:.bs-callout-info}
 You can learn more about using the New Relic APM and Infrastructure agents to collect and analyze application data from the [New Relic APM] and [New Relic Infrastructure][New Relic infrastructure] documentation.
 
-## Monitor performance with alert policies
+### View and analyze log data
+
+After you log in to your New Relic account, you can use the New Relic Logs UI to search across the aggregated log data to troubleshoot application, infrastructure, CDN, and WAF errors. Also, you can connect the log data with other data collected by New Relic APM and Infrastructure services to create charts, dashboards, and alerts to manage Magento application and cloud service operations.
+
+{:.procedure}
+To use the New Relic Logs UI:
+
+1. Use your New Relic credentials to [log in to your New Relic account][New Relic login].
+
+1. From the New Relic ONE home page, select **Logs**.
+
+   ![Cloud project New Relic ONE Logs service]({{ site.baseurl }}/common/images/cloud/cloud-new-relic-one-logs-access.png){:width="650px"}
+
+   If the **Logs** application does not display, you can also access New Relic Logs using search, or navigating from the [APM applications page](#investigate-performance) navigation menu.
+
+1. To review log data for cloud services, enter the `has:"ident"` query in the query field. Then, click **Query logs**.
```

Comment:
This reads oddly because of the query statement. Fast readers will not see the subtle font change, maybe. Could try:

```suggestion
1. To review log data for cloud services, enter a query statement in the _Find logs where_ field. Then, click **Query logs**.
```

Reply:
I like the font clarification suggestion...perhaps they can be merged so that the user doesn't have to try and re-type from the image?  Add:
 Such as `has:"ident"` somewhere to @hguthrie 's suggestion